js/menu.js

```diff
@@ -0,0 +1,32 @@
+function showMenu() {
+  const mobileMenu = document.querySelector('#mobile-menu');
+  const img = document.querySelector('#menu-btn img');
+  const a = document.querySelector('#header a');
+
+  mobileMenu.style.display = 'block';
+  img.src = './imgs/x.svg';
+  a.style.opacity = 0;
+}
+
+function hideMenu() {
+  const mobileMenu = document.querySelector('#mobile-menu');
+  const img = document.querySelector('#menu-btn img');
+  const a = document.querySelector('#header a');
+
+  mobileMenu.style.display = 'none';
+  img.src = './imgs/IconMenu.svg';
+  a.style.opacity = 1;
+}
+
+function toggleHamburgerBtn() { // eslint-disable-line no-unused-vars
```

Comment:
- [ ] It not an acceptable practice to disable linters in your project. Linter where set up in the first place to help you write standard code and disabling it defeats this purpose. Kindly remove the disble linter code on 21.